tools/directivesgen/main.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2023 The OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	_ "embed"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"html"
+	"html/template"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+type Directive struct {
+	Name             string
+	Description      string
+	Syntax           string
+	Default          string
+	Date             string
+	LastModification string
+	Content          string
+}
+
+//go:embed template.md
+var contentTemplate string
+
+const dstDir = "./content/docs/seclang/directives"
+
+func main() {
```

Comment:
This doesn't need to be a main, it can just be an exported function that's called

Reply:
Any advantage on not being main? I think we can keep it and then export a function if we see the need.